setup.py

```diff
@@ -90,6 +90,7 @@ def run(self):
           'absl-py>=0.1.2',
           'google-api-python-client>=1.1',
       ],
+      extra_require={'profiler' : ['memory_profiler']},
```

Comment:
Just question to @mesmith75 didn't we already have an extra_require for LHCb? What happened to that?

Reply:
Its in another PR - #1462

Reply:
thanks